libs/components/src/lib/number-field/number-field.ts

```diff
@@ -46,6 +47,8 @@ function makeStep(element: NumberField, direction: number) {
 @errorText
 @formElements
 export class NumberField extends FastNumberField {
+	@attr({attribute: 'add-button-aria-label'}) addButtonAriaLabel: string | null = null;
+	@attr({attribute: 'subtract-button-aria-label'}) subtractButtonAriaLabel: string | null = null;
```

Comment:
I think, like their label, increment / decrement is a more fitting name
```suggestion
	@attr({attribute: 'increment-button-aria-label'}) addButtonAriaLabel: string | null = null;
	@attr({attribute: 'decrement-button-aria-label'}) subtractButtonAriaLabel: string | null = null;
```